generated-structs.go

```diff
@@ -1,3929 +1,1706 @@
-	/// or <see cref="Page.SetDefaultTimeout"/> methods.
-	/// </para>
-	/// </summary>
+	// Maximum time in milliseconds, defaults to 30 seconds, pass `0` to disable timeout. The default value can be changed by using the BrowserContext.SetDefaultTimeout or Page.SetDefaultTimeout methods.
 	Timeout *float64 `json:"timeout"`
 }
 type FrameDragAndDropOptions struct {
-	/// <summary>
-	/// <para>
-	/// Whether to bypass the <a href="./actionability.md">actionability</a> checks. Defaults
-	/// to <c>false</c>.
-	/// </para>
-	/// </summary>
+	// Whether to bypass the [actionability](./actionability.md) checks. Defaults to `false`.
 	Force *bool `json:"force"`
-	/// <summary>
-	/// <para>
-	/// Actions that initiate navigations are waiting for these navigations to happen and
-	/// for pages to start loading. You can opt out of waiting via setting this flag. You
-	/// would only need this option in the exceptional cases such as navigating to inaccessible
-	/// pages. Defaults to <c>false</c>.
-	/// </para>
-	/// </summary>
+	// Actions that initiate navigations are waiting for these navigations to happen and for pages to start loading. You can opt out of waiting via setting this flag. You would only need this option in the exceptional cases such as navigating to inaccessible pages. Defaults to `false`.
 	NoWaitAfter *bool `json:"noWaitAfter"`
-	/// <summary>
-	/// <para>
-	/// Maximum time in milliseconds, defaults to 30 seconds, pass <c>0</c> to disable timeout.
-	/// The default value can be changed by using the <see cref="BrowserContext.SetDefaultTimeout"/>
-	/// or <see cref="Page.SetDefaultTimeout"/> methods.
-	/// </para>
-	/// </summary>
-	Timeout *float64 `json:"timeout"`
-	/// <summary>
-	/// <para>
-	/// When set, this method only performs the <a href="./actionability.md">actionability</a>
-	/// checks and skips the action. Defaults to <c>false</c>. Useful to wait until the
-	/// element is ready for the action without performing it.
-	/// </para>
-	/// </summary>
+	// Maximum time in milliseconds, defaults to 30 seconds, pass `0` to disable timeout. The default value can be changed by using the BrowserContext.SetDefaultTimeout or Page.SetDefaultTimeout methods.
+	Timeout *float64 `json:"timeout"`
+	// When set, this method only performs the [actionability](./actionability.md) checks and skips the action. Defaults to `false`. Useful to wait until the element is ready for the action without performing it.
 	Trial *bool `json:"trial"`
 }
 type FrameEvalOnSelectorOptions struct {
-	/// <summary><para>Optional argument to pass to <paramref name="expression"/>.</para></summary>
+	// Optional argument to pass to [`param: expression`].
 	Arg interface{} `json:"arg"`
 }
 type FrameEvalOnSelectorAllOptions struct {
-	/// <summary><para>Optional argument to pass to <paramref name="expression"/>.</para></summary>
+	// Optional argument to pass to [`param: expression`].
```

Comment:
this needs to be fixed.

Reply:
Revert back to paramref ?

Reply:
Instead of:
`Optional argument to pass to [`param: expression`].`
It should be:
`Optional argument to pass to `expression`.`

(github does markdown formatting obviously. So remove the `[param: ` stuff that the expression is in an own small code block.

Reply:
This can be followed up